### ISSUE TITLE
Bot is being detected from nike, Making servers unavailable

### ISSUE THREAD

**Issue by TechWithTy** [alexschimpf/Snkrs-Bot#87]
If anyone would like to work toward fixing this issue pleaser reply I'm willing to work with you

**Comment by hi-artem** [comment]
@TyriqueDaniel Any idea how they are detecting the bot? What changes are required?

**Comment by Lazy904** [comment]
Is this an issue still?

**Comment by leoniewgnr** [comment]
Yes, it's still an issue. After the login, this error message appears: 
<img width="519" alt="image" src="https://user-images.githubusercontent.com/42536262/170889071-b9ae031e-290a-400c-bfc1-8f1ea6dbc43a.png">

**Comment by anonymous-sidhax** [comment]
> If anyone would like to work toward fixing this issue pleaser reply I'm willing to work with you

I would love to work on it. What are your suggestions? Maybe rotating UA and proxies
